### ISSUE TITLE
`orthogonal` not in groups

### ISSUE THREAD

**Issue by AnzhiZhang** [JuliaLinearAlgebra/MatrixDepot.jl#124]
hadamard has a groups `orthogonal`, however, this is not listed in the `listgroups` and `data.jl`
https://github.com/JuliaLinearAlgebra/MatrixDepot.jl/blob/cbfd310c5204dc4a720a770c2c3b8ba4102d6ec7/src/higham.jl#L81

**Comment by KlausC** [comment]
That is correct. The group `:orthogonal` is missing. Its only member will be "hadamard".
That will be fixed in an upcoming release.

**Comment by KlausC** [comment]
fixed in v1.0.17
